byte RECONFIGURE_STOPPED = 3
byte RECONFIGURE_OPEN = 1
byte RECONFIGURE_RUNNING = 0
